src/rgw/rgw_kms.cc

```diff
@@ -333,14 +345,14 @@ class KmipSecretEngine: public SecretEngine {
 protected:
   CephContext *cct;
 
-  int send_request(std::string_view key_id, JSONParser* parser) override
-  {
-    return -EINVAL;
-  }
-
-  int decode_secret(JSONObj* json_obj, std::string& actual_key){
-    return -EINVAL;
-  }
+//  int send_request(std::string_view key_id, JSONParser* parser) override
```

Comment:
same here, what's happening with commented code?

Reply:
nit; will remove.

This was "old" code that I left in case I had to put it back in.